trx/fetcher.py

```diff
@@ -21,14 +21,14 @@ def get_testing_files_dict():
     """ Get dictionary linking zip file to their Figshare URL & MD5SUM """
     return {
         'DSI.zip':
-            ('https://figshare.com/ndownloader/files/35617193',
-             '7d7082f7f2e07cb39e2fef13107af169'),
+        ('https://figshare.com/ndownloader/files/37624154',
+         'b847f053fc694d55d935c0be0e5268f7'), # V1 (27.09.2022)
```

Comment:
Is this also supposed to be "V2"?

```suggestion
         'b847f053fc694d55d935c0be0e5268f7'), # V2 (27.09.2022)
```

Reply:
There was no DSI.zip before, so this is the first version.

Reply:
Gotcha. Sounds good!